stackrox/stackrox
# Update Scanner

## Description

Update to latest version of Scanner:

* Fix block alignment for disk overflow when scanning large ELF files
* Other ELF-related fixes
* Add Alpine 3.16 support
* Add manual entry for CVE-2017-5638 to ensure correct data
* Update Alpine vuln link to https://www.cve.org/ instead of https://cve.mitre.org/
* Fix RHSA-2022:4991
* Update nvdtools dependency to fix CVSSv3 equation (correctly calculates a score of 0.0 now)
* Update other various Golang dependencies

## Checklist

## Testing Performed

CI
